spec/spec_helper.rb

```diff
@@ -29,3 +38,19 @@ def verify_concat_fragment_exact_contents(subject, title, expected_lines)
   content = subject.resource('concat::fragment', title).send(:parameters)[:content]
     content.split(/\n/).reject { |line| line =~ /(^#|^$|^\s+#)/ }.should == expected_lines
 end
+
+# See https://github.com/rodjek/rspec-puppet/issues/215
```

Comment:
Not initially no. But when the example count increased the tests started to take a lot of RAM, around 3 GB.